apollo-router/tests/telemetry_test.rs

```diff
@@ -1,55 +0,0 @@
-mod common;
-mod telemetry;
-
-use std::result::Result;
-
-use tower::BoxError;
-
-use crate::common::IntegrationTest;
-use crate::common::Telemetry;
-
-#[ignore]
-#[tokio::test(flavor = "multi_thread")]
-async fn test_otlp_tracing() -> Result<(), BoxError> {
```

Comment:
Is removing rather than moving these deliberate?

Reply:
Yeah, these tests are ignored anyway, and they didn't have anything actually set up to verify. They were just placeholders.